components/02-molecules/menus/main-menu/main-menu.js

```diff
@@ -13,15 +13,15 @@ Drupal.behaviors.mainMenu = {
       });
 
       // Expose mobile sub menu on click.
-      for (let i = 0; i < expandMenu.length; i += 1) {
-        expandMenu[i].addEventListener('click', (e) => {
+      Array.from(expandMenu).forEach((item) => {
```

Comment:
Hmm, what is `expandMenu` in this case? If it's not an array, there should be no need to do any looping. If it _is_ an array, then there's no need to call `Array.from`, you can simply call `expandMenu.forEach()`

Reply:
... or perhaps this is an IE11 weird thing? :smile:

Reply:
It's apparently a dom node thing - creates something called a node list that has to be converted into an array.